rooms.js

```diff
@@ -1478,7 +1478,7 @@ class ChatRoom extends Room {
 		let buffer = '';
 		let counter = 0;
 		for (let i in this.users) {
-			if (!this.users[i].named) {
+			if (!Users(this.users[i]) || !this.users[i].named) {
```

Comment:
...do you understand how garbage collection works?

The user object is named because `.named = false` is not part of the expiration process.

The bug is that expired user objects should not be in the user list in the first place.